Description: man pages: fix hyphen/minus errors
Author: Tzafrir Cohen <tzafrir@debian.org>
Last-Changed: 2010-04-03

Troff uses '-' for a hyphen and \- for a minus sign. Fix some remaining
cases where those two are confused.

--- a/hspell.3
+++ b/hspell.3
@@ -179,12 +179,12 @@ to "/usr/local/share/hspell/hebrew.wgz".
 On most systems, the Hspell library is compiled to use the Zlib library
 for reading the compressed dictionaries. Therefore, a program linking with
 the Hspell library must also be linked with the Zlib library (usually, by
-adding "-lz" to the compilation line).
+adding "\-lz" to the compilation line).
 
 Programs that use
 .I autoconf
 to search for the Hspell library, should remember to tell AC_CHECK_LIB
-to also link with the -lz library when checking for -lhspell.
+to also link with the \-lz library when checking for \-lhspell.
 
 .SH CAVEATS
 While the API described here has been stable for years, it may change
--- a/hspell.1
+++ b/hspell.1
@@ -38,7 +38,7 @@ The output will also be in ISO-8859-8 en
 so it is normally useful to pipe it to bidiv(1) before viewing, as in:
 .PP
 .RS
-.B "hspell -c filename | bidiv | less"
+.B "hspell \-c filename | bidiv | less"
 .RE
 .PP
 If no input file is given,
@@ -162,9 +162,9 @@ This allows conveniently spell-checking
 Running
 .B hspell
 with the program name
-.B hspell-i
+.B hspell\-i
 also enables the
-.B -i
+.B \-i
 option. This is a useful trick when an application expects just the name
 of a spell-checking program, and adds only the "\-a" option (without giving
 the user an option to also add "\-i"). The
